.env

```diff
@@ -1,3 +1,3 @@
 PYTHONUNBUFFERED=true
 SELENIUM_DOCKER_TAG=latest
-
+COMPOSE_PROJECT_NAME=tools
```

Comment:
What's this used for (I don't see it in this diff)?

Reply:
This env var is for docker, it is so that the the volume gets named `tools_awx_db`, instead of `sources_awx_db`